src/components/StatusAlert/index.jsx

```diff
@@ -25,11 +27,13 @@ const StatusAlert = ({
   );
 
   React.useEffect(() => {
-    document.querySelector('html').scrollTo({
-      top: 0,
-      behavior: 'smooth',
-    });
-  }, [hasSubmitted, step]);
+    if (isMounted.current) {
```

Comment:
isn't `useIsMounted` here redundant?

Reply:
goal here is checking if *still* mounted, not if mounted *yet*